src/sacn/receiver_state.c

```diff
@@ -294,6 +291,46 @@ void remove_all_receiver_sockets(socket_close_behavior_t close_behavior)
     remove_receiver_sockets(receiver, close_behavior);
 }
 
+/*
+ * Called in a loop by each receiver thread to manage incoming data and receiver state.
+ */
+void iterate_thread(SacnRecvThreadContext* context)
```

Comment:
I think this function could have a more descriptive name.  It shouldn't be iterate_thread, rather it should be the functionality that the thread keeps calling. read_receiver_messages?

Reply:
I will change this in the next PR.